ruby_on_rails/linting_and_automatic_check.md

```diff
@@ -59,6 +59,8 @@ group :development, :test do
 end
 ```
 
+Use the [reek configuration provided](../templates/.reek) adding a `.reek` file in the root of the project.
```

Comment:
maybe:
``Use the [provided reek configuration](../templates/.reek) by adding a `.reek` file in the root of the project.``